core/src/Http/HttpRequest.cs

```diff
@@ -74,16 +80,39 @@ private static HttpRequestMessage CreateRequestMessage(Uri endpoint, Dictionary<
             return requestMessage;
         }
 
+        private static async Task<HttpContent> CloneContent(HttpContent content)
```

Comment:
>CloneContent [](start = 47, length = 12)

to me this is performing a deep copy and thus naming as such makes more meaning to me... but that might be old fashioned